x/rollapp/transfergenesis/ibc_module.go

```diff
@@ -77,20 +88,20 @@ func (w IBCModule) logger(
 	)
 }
 
-// OnRecvPacket will, if the packet is a transfer packet:
-// if it's not a genesis transfer: enable transfers and pass on the packet. This marks the end of the genesis phase.
-// else:
-//
-//		 transfers must not have already been enabled.
-//	     check it's a valid genesis transfer. If it is, then register the denom
-//
+// OnRecvPacket will,
+// if iro plan exists for this rollapp:
+//	    check it's a valid genesis transfer.
+// 		If it is, then pass the packet, register the denom and settle the plan.
+// In any case, mark the transfers enabled.
```

Comment:
why aren't we actually just setting by default on the rollapp `transfers_enabled=true` and only in the case of IRO plan creation change it to `false` vs "opening" the bridge for regualr transfers, which in essence does nothing.

Reply:
done